src/EditorFeatures/CSharp/SplitStringLiteral/SplitStringLiteralCommandHandler.StringSplitter.cs

```diff
@@ -46,6 +44,17 @@ private abstract class StringSplitter
                 TabSize = tabSize;
                 _indentStyle = indentStyle;
                 CancellationToken = cancellationToken;
+
+                var options = document.GetOptionsAsync().Result;
```

Comment:
SplitStringLiteralCommandHandler already handles getting and using options.  so can we move this to that code and then have it pass in the appropriate newline trivia into the splitter.

Reply:
Done